src/assets/CollectiblesController.ts

```diff
@@ -489,6 +490,13 @@ export class CollectiblesController extends BaseController<
         selectedAddress = detection.userAddress;
       } else {
         chainId = this.config.chainId;
+
```

Comment:
Place this outstide the if/else block, so that the chainId set in the if block is covered as well, in case it is not properly formatted